content/docs/2.5/scalers/newrelic.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "New Relic"
+availability = "2.5+"
+maintainer = "Community"
+description = "Scale applications based on NewRelic NRQL"
+layout = "scaler"
+go_file = "newrelic_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `New Relic` trigger that scales based on a New Relic metric.
+
+```yaml
+triggers:
+  - type: new-relic
+    metadata:
+      # Required: Account - Subaccount to run the query on
+      Account: 1234567
+      # Required: QueryKey - Api key to connect to New Relic
+      QueryKey: "NRAK-xxxxxxxxxxxxxxxxxxxxxxxxxxx"
+      # Optional: nrRegion - Region to query data for
+      Region: "US"
+      # Required: metricName
+      metricName: "duration"
+      # Required: nrql
+      nrql: "SELECT average(duration) from Transaction where appName='SITE' TIMESERIES"
+      # Required: threshold
+      threshold: 100
+      # Optional: nrLogLevel
+      LogLevel: "info"
```

Comment:
We shouldn't set logLevel on scaler level, but this should be inherited from the KEDA operator.

Reply:
Filed a PR to remove the LogLevel for now to get v1 of the scaler in place maybe a bit quicker. Don't disagree with the sentiment. Unclear to me currently how other scalers are actaully handling the logging levels for their client initializers. Don't see anyone pulling info from the ctrl/zap object otherwise. It could be a bit of the operator framework I'm not familar with. A hint would be appreciated if I'm just missing something obvious that would allow for the desired behavior.